template/{{.input_root_dir}}/.gitlab/cd_production.yml.tmpl

```diff
@@ -0,0 +1,15 @@
+image: {{ .input_docker_image }}:latest
```

Comment:
qq, why are these yml.tmpl name convention not consistent with what's in .github and .azure?
e.g in .github this is called
.github/workflows/{{.input_project_name}}-bundle-cd-prod.yml.tmpl

Reply:
+1 it would really help if we can follow similar naming conventions across providers

Reply:
updated to follow the naming conventions